packages/palette/src/elements/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC } from "react"
+import { Box, Flex } from "@artsy/palette"
+import styled, { css } from "styled-components"
+import { zIndex } from "styled-system"
+import { FocusOn } from "react-focus-on"
+
+export interface DrawerProps {
+  open: boolean
+  anchor?: "left" | "right"
+  zIndex?: number
+  onClose?(): void
+}
+
+export const Drawer: FC<DrawerProps> = ({
+  children,
+  anchor = "right",
+  zIndex = DEFAULT_DRAWER_Z_INDEX,
+  open,
+  onClose,
+}) => {
+  const openClass = open ? "open" : ""
+
+  return (
+    <Container zIndex={zIndex} anchor={anchor} className={openClass}>
```

Comment:
No need to pass the classname any longer.

Reply:
done in https://github.com/artsy/palette/pull/1295/commits/0a669fce9351066ec0b313301a2805dea7a9d8fb